src/lib/y2storage/dialogs/inst_disk_proposal.rb

```diff
@@ -0,0 +1,87 @@
+# encoding: utf-8
+
+# Copyright (c) [2016] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may
+# find current contact information at www.novell.com.
+
+require "yast"
+require "y2storage"
+require "ui/installation_dialog"
+
+Yast.import "HTML"
+
+module Y2Storage
+  module Dialogs
+    # Calculates the storage proposal during installation and provides
+    # the user a summary of the storage proposal
+    class InstDiskProposal < ::UI::InstallationDialog
+      # For Devicegraph#actiongraph
+      using Y2Storage::Refinements::Devicegraph
+
+      def initialize
+        super
+        textdomain "storage"
+      end
+
+    protected
+
+      def settings
+        settings = Y2Storage::ProposalSettings.new
```

Comment:
For the time being, yes. https://trello.com/c/9Uj9DmG6/384-3-storageng-installation-storage-proposal

We will process the "connection" of settings with control.xml and with the UI in an upcoming PBI.